src/target_registry.cpp

```diff
@@ -217,8 +242,48 @@ const BitcodeLib *TargetLibRegistry::getISPCTargetLib(ISPCTarget target, TargetO
     return lGetTargetLib(m_targets, target, os, arch);
 }
 
+// Include auto-generated stdlib target map from CMake
+// This map is generated by cmake/StdlibFamilies.cmake based on X86_TARGETS
+// to ensure consistency between build system and runtime lookup
+#include "stdlib_target_map_generated.h"
+
+// Initialize the stdlib target map (generated from CMake)
+static std::unordered_map<ISPCTarget, ISPCTarget> stdlibTargetMap = createStdlibTargetMap();
```

Comment:
Do we really need to define a function to create the map? IMO, it is better to have in the included header the direct declaration of the map.